e2e/test/iothub/service/ExportDevicesTests.cs

```diff
@@ -199,7 +199,7 @@ public async Task DevicesClient_ExportDevices(StorageAuthenticationType storageA
             {
                 await Task.Delay(s_waitDuration).ConfigureAwait(false);
                 jobProperties = await serviceClient.Devices.GetJobAsync(jobProperties.JobId).ConfigureAwait(false);
-                Logger.Trace($"Job {jobProperties.JobId} is {jobProperties.Status} with progress {jobProperties.Progress}% after {sw.Elapsed}.");
```

Comment:
progress seems to be only 0 or 100, so removing from the API surface so the user doesn't look to use it. Instead, they should use status. If progress ever gets interesting, we can make it public.